src/core/midi/MidiPort.cpp

```diff
@@ -169,6 +169,35 @@ void MidiPort::processOutEvent( const MidiEvent& event, const TimePos& time )
 			outEvent.setVelocity( fixedOutputVelocity() );
 		}
 
+		auto activePitch = m_activeMIDIPitch.find(event.key());
+		if( activePitch != m_activeMIDIPitch.end() )
+		{
+			outEvent.setKey( *activePitch );
+			
+			if( event.type() == MidiNoteOff )
+			{
+				m_activeMIDIPitch.remove( event.key() );
+			}
+		}
+		else
+		{
+			int16_t outKey = event.key();
+
+			if( fixedOutputNote() >= 0 &&
+				( event.type() == MidiNoteOn ||
+				  event.type() == MidiNoteOff ||
+				  event.type() == MidiKeyPressure ) )
+			{
+				outKey = fixedOutputNote();
+			}
+
+			if( event.type() == MidiNoteOn || event.type() == MidiKeyPressure )
+			{
+				m_activeMIDIPitch.insert( event.key(), outKey );
```

Comment:
Should call ``insert`` using a ``std::pair<int16_t, int16_t>``, if we are going to be using ``std::unordered_map``.

```suggestion
				m_activeMIDIPitch.insert( {event.key(), outKey} );
```